requirements/test_functional.txt

```diff
@@ -0,0 +1,8 @@
+# sha256: bwGLPxbC1cSQ-u-cot0Pk4LpMPi3gDp1cVGMTDQE78o
+pytest==2.8.1
```

Comment:
We could potentially even combine these and have a tests.txt, what do you think?